web/css/review-form.css

```diff
@@ -8,13 +8,14 @@
 }
 
 .sentence-box {
-  height: 6rem;
+  height: 9.25rem;
```

Comment:
this and the change below in the CSS seem to have negative effects, there is quite some overlap now:

<img width="954" alt="Screenshot 2019-08-14 at 18 38 32" src="https://user-images.githubusercontent.com/330324/63039077-0b9ff800-beb2-11e9-8132-8bf322e07cbe.png">

Reply:
@MichaelKohler I believe its fixed now! I only updated the `validator-height` class that is a css variable. 
I'm not able to test it with more than one sentence as I can't review the sentences I add. Could you guide me to how I could add more mock sentences? Thank you!! 

Let me know what you think! :)